src/infra/main/template.yaml

```diff
@@ -83,33 +88,42 @@ Globals:
         - Fn::ImportValue: !Sub ${VpcStackName}-AWSServicesEndpointSecurityGroupId
 
 Resources:
-  AISSampleLambdaFunction:
+  InterventionProcessorFunction:
     Type: AWS::Serverless::Function
-    # checkov:skip=CKV_AWS_116: DLQ is not appropriate for a Lambda invoked by an API
-    # checkov:skip=CKV_AWS_115: Function is not configured for function-level concurrent execution Limit
     Properties:
-      FunctionName: !Sub '${AWS::StackName}-InterventionHistoryWriterFunction'
-      CodeUri: ../../../dist/handlers/sample-lambda-handler
-      Timeout: 10
+      FunctionName: !Sub "${AWS::StackName}-InterventionProcessorFunction"
```

Comment:
nit: are we going with Intervention or Interventions? Naming differences between the function name and handler for example but more just wondering :)

Reply:
It's a good point... let me have a look overall - our main application name is plural (interventionS), but for this lambda it may make sense either way

Reply:
gone for plural everywhere